helm/charts/kratos/values.yaml

```diff
@@ -324,6 +324,11 @@ statefulSet:
   #    cpu: 100m
   #  memory: 128Mi
 
+  extraArgs: []
+  # -- Array of extra arguments to be passed down to the StatefulSet. Kubernetes args format is expected
```

Comment:
```suggestion
  # -- Array of extra arguments to be passed down to the StatefulSet. Kubernetes args format is expected
  extraArgs: []
```

Reply:
The helm-docs app we use for docs generation requires the comment be above the variable :)